src/RGBColorSpace.js

```diff
@@ -3,19 +3,17 @@ import {multiplyMatrices} from "./util.js";
 import adapt from "./adapt.js";
 import XYZ_D65 from "./spaces/xyz-d65.js";
 
-/**
- * Convenience class for RGB color spaces
- * @extends {ColorSpace}
- */
+// Type "imports"
+/** @typedef {import("./types.js").RGBOptions} RGBOptions */
+
+/** Convenience class for RGB color spaces */
```

Comment:
Why the change in formating and the removal of `@extends`?

Reply:
I removed the `@extends` because TypeScript already knows due to `extends ColorSpace`. The formatting change was just because the JSDoc fit on a single line after removing `@extends`